rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/rest/FightResource.java

```diff
@@ -121,4 +121,30 @@ public FightResource(FightService service) {
 	public String hello() {
 		return "Hello Fight Resource";
 	}
+
+	@GET
+	@Produces(TEXT_PLAIN)
+	@Path("/hello/heroes")
+	@Tag(name = "hello")
+	@Operation(summary = "Ping Heroes hello")
+	@APIResponse(
+		responseCode = "200",
+		description = "Ping Heroes hello"
+	)
+	public String helloHeroes() {
```

Comment:
Should this be `@NonBlocking`?

Reply:
Yeah I don't know the answer either. I'd have to investigate.

Reply:
I just pushed the non-blocking code for ping/heroes. Can you have a look ? Is `Uni.createFrom().item("Hello Hero Resource")` the only way to create a Uni from a String (I looked for something less verbose, but couldn't find it)

Reply:
Yep thats how you create a `Uni` from an item you already have.